binder/src/main/java/io/grpc/binder/internal/BinderTransport.java

```diff
@@ -326,39 +326,30 @@ final void sendSetupTransaction() {
   }
 
   @GuardedBy("this")
-  final void sendSetupTransaction(IBinder iBinder) {
-    Parcel parcel = Parcel.obtain();
-    try {
-      parcel.writeInt(WIRE_FORMAT_VERSION);
-      parcel.writeStrongBinder(incomingBinder);
-      if (!iBinder.transact(SETUP_TRANSPORT, parcel, null, IBinder.FLAG_ONEWAY)) {
-        shutdownInternal(
-            Status.UNAVAILABLE.withDescription("Failed sending SETUP_TRANSPORT transaction"), true);
```

Comment:
Just confirming that this case will become an "INTERNAL" error now.

I think that's ok since we don't expect it, but I also want to be paranoid about how OEMs might have changed edge-case semantics of this API.

Reply:
Yes you are right this will be a small behavior change. 

According to the AOSP source code, the only way BinderProxy.transact() can return false (which we now turn into a RemoteException) is if there's something internally wrong with the iBinder or parcel arguments. Feels like INTERNAL is a more accurate description of this.